src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void setSizeCheck() {
+        int size = numbers.size();
+
+        assertThat(size).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함